### PR TITLE
Worked through Django Starter lesson. Could not get the python manage…

### PR DESCRIPTION
**Pull request by hunnypotpy**
….py merge at the very end to work, i got an import error for views in my urls.py file. Last step before the crud homework.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (DjangoBlog)" project-jdk-type="Python SDK" />
+  <component name="PyCharmProfessionalAdvertiser">
```
Comment (bbelderbos) [inline .idea/misc.xml:4]: ignore the .idea folder = add it to .gitignore then delete it from the git version control cache with `git rm -r .idea`

File: blog/models.py [hunk 2]
```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
 # Create your models here.
+
+class Blog(models.Model):
```
Comment (bbelderbos) [inline blog/models.py:5]: nice, seems you did not make and/or commit the migration file, I did it for you quickly pushing the change to this branch:

image

File: blog/models.py [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+    title = models.CharField(max_length=100)
```
Comment (bbelderbos) [inline blog/models.py:6]: could add `unique=True` (would require another migration) to let the DB enforce unique blog post titles

File: blog/forms.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from django.forms import ModelForm
+
+from .models import Blog
+
+class BlogForm(ModelForm):
```
Comment (bbelderbos) [inline blog/forms.py:5]: nice, Django forms will save you code! :)

**Review by bbelderbos — APPROVED** [review]
Good stuff, so onto the views now using the form right?

**Comment by bbelderbos** [comment]
Let me know if you need more help ...

File: .idea/misc.xml (hunk 1 above)
Comment (hunnypotpy) [reply .idea/misc.xml:4]: Ok, I can do that! :) Is that the next video in the Django tutorial?

File: blog/models.py (hunk 2 above)
Comment (hunnypotpy) [reply blog/models.py:5]: Is that the 0001_initial.py file that sits in the migration folder? If so, I have it, I wonder why it won't commit though.

File: blog/models.py (hunk 3 above)
Comment (hunnypotpy) [reply blog/models.py:6]: I like that idea! Where would that line of code go?

File: blog/forms.py (hunk 4 above)
Comment (hunnypotpy) [reply blog/forms.py:5]: Whoo hooo! :)

File: .idea/misc.xml (hunk 1 above)
Comment (bbelderbos) [reply .idea/misc.xml:4]: no, this is related to source control and what to ignore, the .idea folder is related to your PyCharm / your IDE so you would not commit that to version control / GitHub

File: blog/models.py (hunk 2 above)
Comment (bbelderbos) [reply blog/models.py:5]: yep that one, hm weird? so it did not show up with `git status`?

File: blog/models.py (hunk 3 above)
Comment (bbelderbos) [reply blog/models.py:6]: `title = models.CharField(max_length=100, unique=True)`

Comment (bbelderbos) [reply blog/models.py:6]: ... then run `python manage.py makemigrations` to make the corresponding migration file (and `git add` it to commit), then run `python manage.py migrate` to sync that change to the DB

**Pull request merged**